.github/workflows/test.yaml

```diff
@@ -20,14 +20,14 @@ jobs:
           architecture: x64
           python-version: ${{ matrix.python }}
 
-      - name: Install Poetry
-        run: pip install poetry
+      - name: Install uv
+        run: pip install uv
 
-      - name: Install Python dependencies
-        run: poetry install
+      - name: Create venv and install Python dependencies from uv.lock
+        run: uv sync
```

Comment:
I left this as `uv sync` rather than `uv sync --python 3.10` to account for future parallel testing of other python versions via the `matrix` and `python: ["3.10"]` later down the road.